src/tool/subcommands/api_cmd/test_snapshot.rs

```diff
@@ -220,4 +221,33 @@ mod tests {
             println!("  succeeded.");
         }
     }
+
+    #[tokio::test]
+    async fn rpc_regression_tests_print_uncovered() {
+        let pattern = lazy_regex::regex!(r#"^(?P<name>filecoin_.+)_\d+\.rpcsnap\.json\.zst$"#);
+        let covered = HashSet::from_iter(
+            include_str!("test_snapshots.txt")
+                .trim()
+                .split("\n")
+                .map(|i| {
+                    let captures = pattern.captures(i).expect("pattern capture failure");
+                    captures
+                        .name("name")
+                        .expect("no named capture group")
+                        .as_str()
+                        .replace("_", ".")
+                        .to_lowercase()
+                }),
+        );
+        macro_rules! print_uncovered {
+            ($ty:ty) => {
+                let name = <$ty>::NAME.to_lowercase();
+                if !covered.contains(&name) {
+                    println!("{} is uncovered.", <$ty>::NAME);
+                }
+            };
+        }
+
+        crate::for_each_rpc_method!(print_uncovered);
```

Comment:
We can change this to throw an error once we have full coverage

Reply:
I don't think it'll ever be empty, maybe it makes sense to implement this as part of, e.g., `forest-tool test` or some subcommand there in `api`.